parcels/fieldset.py

```diff
@@ -136,7 +134,7 @@ def from_data(
             lat = dims["lat"]
             depth = np.zeros(1, dtype=np.float32) if "depth" not in dims else dims["depth"]
             time = np.zeros(1, dtype=np.float64) if "time" not in dims else dims["time"]
-            time = np.array(time) if not isinstance(time, np.ndarray) else time
+            time = np.array(time)
```

Comment:
So I assume this is a valid change because calling `np.array()` on an `np.array` doesn't change anything to the array?

Reply:
yes :)